docs/plugins/composable_resources.md

```diff
@@ -132,6 +132,8 @@ Getting files from the replication resource will show a preference for locality.
 
 #### num_repl and read keywords
 
+NOTE: The num_repl keyword was deprecated in 4.2.8.
```

Comment:
backticks around `num_repl`... and and say 'and will be removed in 4.3.0'

Reply:
Done!